site/profile/manifests/cvmfs.pp

```diff
@@ -118,4 +128,8 @@
   # 'use_fusefs_home_dirs' policy fix that issue.
   selinux::boolean { 'use_fusefs_home_dirs': }
 
+# Creates an alien cache repositories if not present
+  ensure_resource('file', 'etc/cvfms/config.d', {'ensure' => 'directory', content => epp('profile/cvmfs/config.d', {
```

Comment:
This resource has been previously defined already.

Reply:
Removed